frontends/p4/actionsInlining.h

```diff
@@ -51,9 +51,7 @@ class ActionsInliner : public AbstractInliner<ActionsInlineList, AInlineWorkList
     std::map<const IR::MethodCallStatement *, const IR::P4Action *> *replMap;
 
  public:
-    explicit ActionsInliner(bool isv1) : refMap(new P4::ReferenceMap()), replMap(nullptr) {
-        refMap->setIsV1(isv1);
```

Comment:
Any reason why you are removing the isv1 checks? I think we should deprecated that strange check eventually but it seems like you may change some semantics here.

Reply:
I think this is because the ReferenceMap being passed in already has the isv1 flag set appropriately.

Reply:
Ah yeah, makes sense.

Reply:
Yes, that's correct.